### ISSUE TITLE
Make "quality levels" more explicit

### ISSUE THREAD

**Issue by danfowler** [openspending/fiscal-data-package#95]
Currently, there is a concept of Fiscal Data Package having "quality" levels.

> We also emphasize that what we provide is a framework rather than a strict standard. That is, we provide recommendations on what information should be provided rather than strict requirements. These recommendations are also categorised into “quality” levels. Each level requires more information be provided.

However, the spec could lay out a more explicit definition of how these quality levels (and parenthetical quality levels) are actually generated from the information provided in the data package.  It might also be helpful to describe how the "quality" level can actually be used by an implementation.

See: https://github.com/openspending/fiscal-data-package/commit/bc3ac9fc0685837414573c9e0efccc8966dabe15#commitcomment-14468305

**Comment by timgdavies** [comment]
In my experience with [IATI](http://www.iatistandard.org) and [OCDS](http://standard.open-contracting.org) a full quality assessment framework is hard to tie down to whether or not certain fields are included. 

The approach we've taken in OCDS is to group kinds of data as 'basic', 'intermediate' and 'advanced', and we're thinking about how to convert that into detailed tests at the field level which a validator could check for - but we're finding that assessing data requires thinking about what it makes sense for a publisher to provide given their level of government, political system and so-on.

Essentially, I think it's worth considering how to build quality levels into guidance, tools and social processes associated with the spec, rather than too tightly into the spec itself.

**Comment by pwalsh** [comment]
IMO, quality stuff is a great example of things to mark for future consideration, and take out of our present scope of work completely.

**Comment by danfowler** [comment]
@pwalsh I agree.

**Comment by pwalsh** [comment]
@danfowler @rgrp 

I actually want to suggest the opposite - we don't make quality levels more explicit - we remove it entirely from the spec. 

Things from the spec like:

> For example, if your aggregated expenditure data has a dimension named “cofog” described as above, it will have a quality level of “2”. Quality levels in parentheses are applied to dimensions which are desirable but not required to reach that quality level.

Are very, very hard to parse and act on (in code or whatever). My guess is that the meaning of this is not known to anyone except the author.

Concretely, I suggest:

1. Completely remove all quality references from the spec
2. Open a new issue to propose a quality section of the spec, and only re-introduce it "later", like v0.5 or above.

WDYT @pudo @akariv @stevage ?

**Comment by rgrp** [comment]
AGREED. Remove from spec for now. May be something in commentary.
